data/human/transport missions.txt

```diff
@@ -3582,3 +3582,714 @@ mission "Hauler VI cargo prototype 3"
+					goto nocando
+				`	"Did he SEE us?"`
+					goto seeus
+			`	"I don't know. I think we oughta get going," says Bunker, looking worried.`
+			choice
+				`	"That's an order!"`
+				`	"OK. Call the drones back."`
+					goto calldrones
+			`	"You're the captain," he responds giving up. You continue to watch the display. The last of the slaves are loaded followed by the security team. The two men are last to get to their ships. You notice a slight hum from outside before getting raked with fire from one of the pirate escorts. You try pull out to space but another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky, exploding on impact.`
+				die
+			label seeus
+			`	"I have no idea! Right now we have bigger things to worry about."`
+			choice
+				`	"Call back the drones immediately."`
+					goto escape
+				`	"I'm sorry but we have to leave the drones behind."`
+					goto nocando
+			label nocando
+			`	Bunker hits the recall button for the drones anyways. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the fighters aren't coming straight for us. We have time. The drones are less than two minutes out."`
+			choice
+				`	"We're leaving. Now!"`
+				`	"Fine. Get to the cargo bay lock down the drones. We take off on your mark."`
+					goto escape
+			`	"I can't let you do that <first>," says Bunker stepping in between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
+			choice
+				`	(Pull your blaster and charge him with mutiny.)`
+					goto blaster
+				`	(Let him go to the cargo bay and lock down the drones.)`
+					goto escape
+			label calldrones
+			`	"Aye aye captain," he says as he hits a button. "Drone ETA, two minutes, fifteen seconds."`
+			`	"Get to the cargo bay," you say. "Lock down the drones and get belted in. We will take off on your mark."`
+			label escape
+			`	"Less than a minute later you hear Bunker on comms, "Drones are locked down and ready to go."`
+			`	The moment you pull out of the canyon pirate fighters appear on your scanners. You accelerate into space at full speed with the fighters in hot pursuit.`
+				launch
+			label blaster
+			`	You pull your blaster and order him to stand down. "Don't do this," he says. You're looking for the handcuffs when you notice a hum coming from outside. Suddenly you are raked with fire from one of the pirate escorts. You drop your blaster and try to lift off into space but another another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky and explodes on impact.`
+				explode
+
+	npc
+		personality heroic nemesis
+		government "Pirate"
+		ship "Quicksilver" "Don't Run"
+		ship "Quicksilver" "Obey"
+		ship "Quicksilver" "Work is Freedom"
+		ship "Quicksilver" "Falange"
```

Comment:
A google search told me that "Falange" is the name of a former spanish political party. The name doesn't seem to fit with the rest so maybe it should be changed.

Reply:
changed it